src/commands/updateComponents.ts

```diff
@@ -0,0 +1,23 @@
+import * as vscode from 'vscode';
```

Comment:
As this is more of a util rather than a vscode command we expose I don't think it should live under the `commands` folder